File: codewars/python/snippets.go
package main

import (
	"fmt"
)

func BouncingBall(h, bounce, window float64) int {
	if (h <= 0) || (window >= h) || (bounce <= 0) || (bounce >= 1) {
		return -1 // your code
	}
	for j := 0; h > window; {
		j++
		h *= bounce
		if h > window {
			j++
		} else {
			return j
		}
	}
	return -1
}
package main

import "fmt"

func create2DArray(n int) [][]int {
    // Erstelle ein 2D-Array mit n Zeilen und n Spalten, gefüllt mit Nullen
    array := make([][]int, n)
    for i := range array {
        array[i] = make([]int, n)
    }
    return array
}

// https://www.codewars.com/kata/534d2f5b5371ecf8d2000a08/train/go
// Multiplication table


func MultiplicationTable(size int) [][]int {
	// Implement me! :)
}

func main() {
    n := 3 // Beispielwert für n
    result := create2DArray(n)
    
    // Ausgabe des erstellten 2D-Arrays
    for _, row := range result {
        fmt.Println(row)
    }
}

File: aoc/2023/Day 11/snippets.py
a = [3,0]
b = [7,8]

sa, sb = (a[0], b[0]) if a[0] < b[0] else (b[0], a[0])

koords = []

for s in range(sa+1, sb+1):
    koords.append((s,a[1]))
    
print(koords)

ea, eb = (a[1], b[1]) if a[1] < b[1] else (b[1], a[1])

for s in range(ea+1, eb+1):
    koords.append((b[0],s))

print(koords,len(koords))
    